addons/compat_ace_fire/XEH_preInit.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+ADDON = false;
+
+GVAR(aceMedicalLoaded) = isClass(configFile >> "CfgPatches" >> "ace_medical_engine");
+if (isClass(configFile >> "CfgPatches" >> "ace_medical") && {!GVAR(aceMedicalLoaded)}) exitWith {
+    INFO("PreInit: Disabled --> old ACE medical loaded");
+};
+
+#include "XEH_PREP.hpp"
+call compile preprocessFileLineNumbers format["%1\XEH_preInit.sqf",REPLACEPATH];
```

Comment:
Why do you need to run the ace fire pre init here? Shouldn't it run on its own?

Reply:
The config.cpp replaces ace fire's preInit and preStart, and this is specifically replacing a function that would otherwise be locked by cba_compile_function in ace's preStart.
https://github.com/CBATeam/CBA_A3/issues/788#issuecomment-339191918